src/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -179,6 +179,7 @@ class StockMovement {
                 driverName: shipment?.driverName,
                 trackingNumber: trackingNumber?.identifier,
                 currentStatus: shipment?.currentStatus,
+                stocklist: requisition?.requisitionTemplate
```

Comment:
@jmiranda on the frontend we are using "stockList", so I think we should rename this field to "stockList" and remove the second field or rename it on forntend to "stocklist". Having two fields will be very confusing and can case issues, also on frontend only stocklist name and id are used, so there is no need to send entire requisition (we can save some transfer).

Reply:
That's just to create the object, but yes it should probably look the same as the JSON. The JSON only returns the ID and name. 
```stockList: [id: stocklist?.id, name: stocklist?.name]```

I would prefer `stocklist` over `stockList` but will change it for now.

Reply:
@pmuchowski Actually there are a bunch of references to `stocklist` throughout the code and since I prefer `stocklist`, I'll just refactor the JSON and UI later.